### PR TITLE
Regression test fixes

### PR DESCRIPTION
**Pull request by clumens**
If needed, I can track down when the second two patches became necessary.  However it takes a fair bit of bisecting to check things against main which is why I didn't do this originally.  I don't think any of these are required on 2.1.

### CODE REVIEW

**Comment by clumens** [comment]
Confirmed - 2.1 passes without patching.

File: cts/cts-exec.in [hunk 1]
```diff
@@ -499,7 +499,7 @@ class ExecTests(Tests):
                      + self._action_timeout)
 
         test.add_cmd(args='-l "NEW_EVENT event_type:exec_complete rsc_id:stonith_test_rsc action:monitor rc:error op_status:error" -t 15000',
-                     kill="killall -9 -q pacemaker-fenced lt-pacemaker-fenced")
+                     kill="killall -9 -q pacemaker-fenced lt-pacemaker-fenced", expected_exitcode=ExitStatus.TIMEOUT)
```
Comment (kgaillot) [inline cts/cts-exec.in:502]: It seems odd that we have rc:error op_status:error here but get a timeout. It's trying to run a monitor command after killing the fencer, so either one could make sense, but the inconsistency is odd.

File: cts/cts-exec.in [hunk 2]
```diff
@@ -205,68 +205,68 @@ class ExecTests(Tests):
 
         self._common_cmds = {
             "ocf_reg_line": '-c register_rsc -r ocf_test_rsc ' + self._action_timeout + ' -C ocf -P pacemaker -T Dummy',
-            "ocf_reg_event": '-l "NEW_EVENT event_type:register rsc_id:ocf_test_rsc action:none rc:ok op_status:complete"',
+            "ocf_reg_event": '-l "NEW_EVENT event_type:register rsc_id:ocf_test_rsc action:none rc:ok op_status:Done"',
```
Comment (kgaillot) [inline cts/cts-exec.in:208]: Doh, I thought I did this but it was just the cli tests :/

File: python/pacemaker/_cts/corosync.py [hunk 3]
```diff
@@ -136,7 +136,7 @@ def start(self, kill_first=False, timeout=10):
         if self.verbose:
             print("Starting corosync")
 
-        with subprocess.Popen("corosync", stdout=subprocess.PIPE) as test:
+        with subprocess.Popen("corosync", stdout=subprocess.PIPE, stderr=subprocess.PIPE) as test:
```
Comment (kgaillot) [inline python/pacemaker/_cts/corosync.py:139]: Yay, I noticed this but didn't have time to track it down

File: cts/cts-exec.in (hunk 1 above)
Comment (clumens) [reply cts/cts-exec.in:502]: I thought this was odd too.  I can work on bisecting to figure out where this broke.

Comment (clumens) [reply cts/cts-exec.in:502]: Hah, figured it out.  This patch is completely unnecessary and the real fix is to change a couple "Error" into "Error occurred".

**Review by kgaillot — APPROVED** [review]

**Pull request merged**